hedera-node/src/main/java/com/hedera/services/txns/token/process/Creation.java

```diff
@@ -113,6 +114,15 @@ public void loadModelsWith(
 			final var autoRenewId = new Id(autoRenewGrpc.getShardNum(), autoRenewGrpc.getRealmNum(), autoRenewNum);
 			autoRenew = accountStore.loadAccountOrFailWith(autoRenewId, INVALID_AUTORENEW_ACCOUNT);
 		}
+//		final var customFeeList = op.getCustomFeesList();
+//		for (var fee : customFeeList) {
+//			if (fee.hasFeeCollectorAccountId()) {
+//				final var feeCollector = fee.getFeeCollectorAccountId();
+//				final var feeCollectorNum = accountStore.getAccountNumFromAlias(feeCollector.getAlias(), feeCollector.getAccountNum());
+//				final var feeCollectorId = new Id(feeCollector.getShardNum(), feeCollector.getRealmNum(), feeCollectorNum);
+//				accountStore.loadAccountOrFailWith(feeCollectorId, INVALID_ACCOUNT_ID);
+//			}
+//		}
```

Comment:
can these be removed?

Reply:
yup, can be removed. The feeCollector validation is actually already done in sigRequirements.